### DISCUSSION TITLE
A homebrew idea non geode related

### DISCUSSION THREAD

**Discussion by ghost** [geode-sdk/sdk#1168] [General]
What if someone ported Geometry Dash into a homebrew game for PS3? The preferred place for the homebrew in my thoughts should go to PSXPlace.

**Comment by OmgRod** [comment]
That's nearly impossible to do. You would need to get the source code of GD which isnt public, then you would have to use a PS3 homebrew SDK to make a homebrew app. The only way it could be possible was if you decompiled GD but that would be time-consuming and unstable.

**Reply by ghost** [reply]
Well one could always start off easy and put early levels first before public release and then make their own Geometry Dash homebrew instead of *porting* if that works better.

**Comment by antonfdiaz** [comment]
You could do a recreation, but a direct port is very hard to make, as @OmgRod said.
